vagrant/packer/template.json5

```diff
@@ -0,0 +1,193 @@
+{
+	variables: {
+		name: "FreeBSD-12.0-CURRENT",
+		hostname: "freebsd",
+
+		cpus: "1",
+		memory: "512",
+		root_volume_size: "16384",
+
+		iso_url: "{{ env `ISO_URL` }}",
+		iso_md5: "{{ env `ISO_MD5` }}",
+	},
+	builders: [{
+		type: "vmware-iso",
+		name: "{{ user `name` }}-VMWare",
+
+		iso_url: "{{ user `iso_url` }}",
+		iso_checksum: "{{ user `iso_md5` }}",
+		iso_checksum_type: "md5",
+
+		disk_size: "{{ user `root_volume_size` }}",
+		
+		guest_os_type: "freebsd-64",
+		shutdown_command: "poweroff; poweroff",
```

Comment:
Why not just `sync && poweroff`?  It smells like there's a story behind the double `poweroff`.

Reply:
There is - and I was hoping you could tell me what it is :-) - this came from your original Packer script for FreeBSD!